---
layout: page
permalink: /research/
title: research
description:
nav: true
---

### **One Sentence Summary**

My research interest is to develop causal inference methods to better understand the genetic architecture of complex diseases.

### **Projects**

- **MA-FOCUS**
TBD
<!-- We developed a Bayesian gene fine-mapping method (MA-FOCUS) of complex diseases leveraging GWAS and gene expression data from multiple ancestries.  -->

- **SuShiE**
<!-- Developed a Bayesian stepwise selection approach (SuShiE) by modelling the covariance structures of SNPs’ effect sizes across multiple ancestires to identify shared causal genetic variants of complex diseases. -->
TBD

- **rPWAS**
TBD
<!-- Developed an integrative approach (rPWAS) leveraging GWAS and protein abundance data to test post-transcriptional/translational regulatory mechanisms of complex diseases. -->

- **Perturb-seq&eQTL**
TBD
<!-- Developed a statistical framework to integrate experimental perturb-seq data for trans effects with GWAS and eQTL data for cis effects to identify “core” genes that are the trans targets of disease-relevant genomic variations. -->


### **Software**

- **[TWAS Simulator](https://github.com/mancusolab/twas_sim){:target="_blank"}** - A Python software to perform Transcriptome-wide Association Studies (TWAS) simulations. Users can choose different prediction methods to fit eQTL models, such as LASSO, ENET, and GBLUP.

- **[MA-FOCUS](https://github.com/mancusolab/ma-focus){:target="_blank"}** - A Python software to implement multi-ancestry gene-trait association fine-mapping.

- **ADNI3**, **LEADS** - R Shiny apps to build interactive tabulation and visualization clinical reports for enrollment, outcome and safety monitoring.

- **ADNI3**, **A4**, **LEADS** - R packages to crawl raw clinical data from EDC API, convert to CDISC format, and generate daily statistical reports.