---
layout: page
permalink: /research/
title: research
description:
nav: true
---

### **One Sentence Summary**

My research interest is to develop statistical methods leveraging summary-based data (e.g. GWAS) and molecular reference panels from multiple ethnic groups to better understand the genetic architecture for complex diseases.

<!-- ### **Two+ Sentences Summary**

On a molecular level, how does genetic information affect phenotypes? Central dogma of molecular biology states that there exists a flow such that DNA makes RNA, RNA makes proteins, proteins function together to support cellular activities (metabolism), and then its products, metabolites, as the final molecular step to affect traits. As scientific technology advances, people are able to measure individuals’ genome, transcriptome, proteome, and metabolome at academically affordable costs. This enables researchers to use statistical methods to explore the association between molecular products and complex traits.

Genome-wide Association Studies (GWAS) regressing traits value on genotypes have identified tons of genetic variants associated with complex traits. However, the biological mechanism of how these variants have an impact on traits remains unknown. The fact that most GWAS hits are located in non-coding regions ground the hypothesis that SNPs influence complex traits through gene expressions (mRNA, proteins etc.). Transcriptome-wide Association Studies (TWAS) that leverages predicted gene expression imputed from external reference panels and GWAS summary statistics have showcased thousands of gene-trait associations whose molecular mechanism is driven by mRNA levels.

However, the full picture of the genetic architecture isn’t clear due to several restrictions. One is that mRNA mediation only explains a small portion of the total variation of complex traits, suggesting that downstream molecular products proteins and metabolites should have independent influences. Another is that genetic architecture may be shared, partially shared, or independent across different ethnic groups, but current GWAS panels are European-ancestry dominated, which hinders biological findings.

Therefore, my PhD research is going to focus on relieving these two limitations. I’m aiming to develop methods that integrate data from genomics, transcriptomics, proteomics, and metabolomics across different populations so that people can have a more complete picture of how genetic information drives complex traits. -->


### **Projects**

- **rPWAS**

  TBD

- **ME-FOCUS**

  TBD

### **Software**

- [**FOCUS**](https://github.com/mancusolab/focus) - a Python software to implement multi-ethnic gene-trait association fine-mapping.

- **ADNI3**, **LEADS** - R Shiny apps to build interactive tabulation and visualization clinical reports for enrollment, outcome and safety monitoring.

- **ADNI3**, **A4**, **LEADS** - R packages to crawl raw clinical data from EDC API, convert to CDISC format, and generate daily statistical reports.
